guides/doc-Release_Notes/topics/katello-4.8.0.adoc

```diff
@@ -0,0 +1,122 @@
+= 4.8.0 Phoenix (2023-03-28)
+
+== Features
+
+=== Hosts
+ * Add support for Erratum release date in Host - Applied Errata report template (https://projects.theforeman.org/issues/36049[#36049], https://github.com/Katello/katello/commit/d1b4c553b19d3056dc5ca5cb133e880b51df638f[d1b4c553])
+
+=== Inter Server Sync
+ * Need incremental export for syncable format (https://projects.theforeman.org/issues/35948[#35948], https://github.com/Katello/katello/commit/3dd58df7579341b0b1dba7017b2e7086cd6477bd[3dd58df7])
+
+=== Content Views
+ * Backend work: As a user I want to be able to assign multiple content views on a content host (https://projects.theforeman.org/issues/35580[#35580], https://github.com/Katello/katello/commit/c33da7a3fa9473add44a154feba4e57f34b2b289[c33da7a3])
+
+=== Tests
+ * ouia-ID for tile cards in the new host details page (https://projects.theforeman.org/issues/35411[#35411], https://github.com/Katello/katello/commit/2c01e818765d49bb38562e9313a4587df18faaeb[2c01e818], https://github.com/Katello/katello/commit/ecd11019cf5c30c1779a756cb116f506ee71e139[ecd11019])
+
+=== Other
+ * Allow installed_debs method (https://projects.theforeman.org/issues/35886[#35886], https://github.com/Katello/katello/commit/e630a6e28a23795739b05c9625747fa8784884c2[e630a6e2])
+ * Last checkin and Registered columns should show up as empty if there's no data (https://projects.theforeman.org/issues/35854[#35854], https://github.com/Katello/katello/commit/3920e5a7384e93769dfa830b8097ed210c9a0e3d[3920e5a7])
+
+== Bug Fixes
+
+=== Inter Server Sync
+ * Importing incremental content not recreating metadata properly (https://projects.theforeman.org/issues/36164[#36164], https://github.com/Katello/katello/commit/df3b102e512c193b213ecc3a8c8ed2f2ef1ef8e6[df3b102e])
+ * Need a better warning message for empty incremental export (https://projects.theforeman.org/issues/36146[#36146], https://github.com/Katello/katello/commit/dde5565960ea6183f35a3aa10efe046b9636c5ae[dde55659])
+
+=== Repositories
+ * Need to update Recommended Repositories page with Satellite 6.13 repos (https://projects.theforeman.org/issues/36158[#36158], https://github.com/Katello/katello/commit/48694791d2922b52c39780529b812231090be9f5[48694791])
```

Comment:
Not a blocker, but as a general guide I'd recommend removing mentions of Satellite and convert them to upstream friendly names. Something we should also keep reminding our Red Hat colleagues of.